identity/src/keypair.rs

```diff
@@ -446,8 +442,7 @@ impl PublicKey {
 
     #[cfg(feature = "ed25519")]
     #[deprecated(
-        since = "0.2.0",
-        note = "This method name does not follow Rust naming conventions, use `PublicKey::try_into_ed25519` instead."
+        note = "This method name does not follow Rust naming conventions, use `PublicKey::from` instead."
```

Comment:
Please revert this. The `into_` function is indeed correctly replaced by `try_into_` :)

Reply:
Whoops, yes!